lib/stale.js

```diff
@@ -10,28 +10,54 @@ module.exports = class Stale {
 
     await this.ensureStaleLabelExists();
 
-    this.github.paginate(this.getStaleIssues(), data => {
-      data.items.filter(issue => !issue.locked)
-                .forEach(issue => this.mark(issue));
-    });
+    this.mark();
+    this.sweep();
```

Comment:
It would seem to me that we would want `mark` to complete before `sweep` begins, but I believe that is currently not enforced in master (and it remains unenforced here).